data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
@Zbizu it's client sided , if the option doesn't pop up when you right click, its because its the wrong id, with lucky otclient is up to date with 10.92, then you could take look there what items has this flag or attribute, I don't know otclient project very well.